README.md

```diff
@@ -131,3 +123,30 @@ To extract metadata from an affiliation string:
 	$ curl -X POST --data "affiliation=the text of the affiliation" \
 	  http://cermine.ceon.pl/parse.do
 
+
+(OPTIONAL) Build & Run from sources
+-------------
+
+If you would like to build an executable JAR yourself, clone the project and follow these steps.
+
+**Preconditions**
+
+- Ensure that Apache Maven is installed (`sudo apt install maven`)
+
+**Build **
+
+```
+$ cd CERMINE/cermine-impl
+$ mvn clean compile assembly:single
+```
+
+Afterwards a new `target` directory is placed within `CERMINE/cermine-impl/` containing a `cermine-impl-<VERSION>-jar-with-dependencies.jar` file. Ensure the file is readable by `java`.
+```
+$ chmod u+x target/cermine-impl-<VERSION>-jar-with-dependencies.jar
```

Comment:
I don't think you need the JAR file to have execute permission. See for example https://stackoverflow.com/questions/2668796/what-are-the-minimum-unix-permissions-to-run-an-executable-jar-file